dockstore-integration-testing/src/test/java/io/dockstore/client/cli/OrganizationIT.java

```diff
@@ -1683,6 +1691,66 @@ public void testDuplicateAliasOperations() {
         }
     }
 
+    /**
+     * Tests that we are getting the number of workflows and tools correctly
+     */
+    @Test
+    public void testWorkflowsToolsLength() {
+        // Setup user who creates Organization and collection
+        final ApiClient webClientUser2 = getWebClient(USER_2_USERNAME, testingPostgres);
+        OrganizationsApi organizationsApi = new OrganizationsApi(webClientUser2);
+
+        final ApiClient webClientAdminUser = getWebClient(ADMIN_USERNAME, testingPostgres);
+        OrganizationsApi organizationsApiAdmin = new OrganizationsApi(webClientAdminUser);
+
+        // Create the Organization and collection
+        Organization organization = createOrg(organizationsApi);
+        Collection stubCollection = stubCollectionObject();
+
+        long orgId = organization.getId();
+
+        // Attach collections
+        Collection collection = organizationsApi.createCollection(orgId, stubCollection);
+
+        long collectionId = collection.getId();
+
+        // Approve the org
+        organization = organizationsApiAdmin.approveOrganization(organization.getId());
+
+        // Publish a tool
+        long entryId = 2;
+        ContainersApi containersApi = new ContainersApi(webClientUser2);
+        PublishRequest publishRequest = SwaggerUtility.createPublishRequest(true);
+        containersApi.publish(entryId, publishRequest);
+
+        // Add tool to collection
+        organizationsApi.addEntryToCollection(orgId, collectionId, entryId, null);
+
+        //manually register a workflow
+        //WorkflowsApi workflowsApi = new WorkflowsApi(webClientUser2);
+        //workflowsApi.manualRegister("GITHUB", "DockstoreTestUser/dockstore-whalesay-wdl", "/dockstore.wdl", "test", "WDL", "");
+        //Workflow workflow = workflowsApi.manualRegister(SourceControl.BITBUCKET.name(), "dockstore_testuser2/dockstore-workflow", "/dockstore.wdl", "",
+        //         DescriptorLanguage.WDL.getShortName(), "");
+        //workflow = workflowsApi.refresh(workflow.getId(), false);
+        //System.out.println(workflow.getId());
+        //workflow = workflowsApi.publish(workflow.getId(), SwaggerUtility.createPublishRequest(true));
```

Comment:
It tells me the repo does not meet the requirement to publish

Reply:
check if it's valid